api/SVGTransformList.json

```diff
@@ -58,7 +58,7 @@
               "version_added": true
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "79"
```

Comment:
`length` property, but lots of the members of `SVGTransformList` are missing, notably including `numberOfItems`. I think this is still correct though.